cfme/tests/control/test_bugs.py

```diff
@@ -181,6 +182,20 @@ def setup_disk_usage_alert(appliance):
     assert not result.failed
 
 
+@pytest.fixture
+def action_for_testing(appliance):
+    action_ = appliance.collections.actions.create(
+        fauxfactory.gen_alphanumeric(),
+        action_type="Tag",
+        action_values={"tag": ("My Company Tags", "Department", "Accounting")}
+    )
+    yield action_
+    view = action_.create_view(EditActionView, wait="10s")
+    view.cancel_button.click()
```

Comment:
I think this cleanup should be a bit safer - check that the view is displayed, or at least that the button is displayed before trying to click it. Its very possible when pytest returns for these statements, the view won't be present and we'll hit exceptions in teardown.

Reply:
After discussing this a bit offline, we've decided to move this cleanup (clicking the `cancel` button) back to the test itself, so that the fixture can be used in future tests.